packages/presentation/src/file.ts

```diff
@@ -284,3 +284,18 @@ async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: str
     })
   }
 }
+
+export async function fetchJson (file: string, name: string): Promise<Blob | undefined> {
+  const resp = await fetch(getFileUrl(file, name), { signal: AbortSignal.timeout(5 * 1000) })
+  if (!resp.ok) {
+    console.error({ error: `failed to process request: ${resp.status}` })
```

Comment:
let's throw an error or do nothing instead of writing to console

Reply:
Thanks, fixed.